<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>243</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(34, 34, 34);
color: rgb(226, 226, 226);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QRadioButton" name="rockRadioButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Rock</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="paperRadioButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Paper</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="scissorsRadioButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>110</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Scissors</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SubmitBtn">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>160</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Sumbit</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>60</y>
      <width>100</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>You Chose:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="playerChoiceLabel">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>60</y>
      <width>100</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>100</y>
      <width>100</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Program Chose:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="programChoiceLabel">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>100</y>
      <width>100</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>160</y>
      <width>100</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Result:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="resultLabel">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>160</y>
      <width>100</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>20</y>
      <width>110</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Player Win Count</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>20</y>
      <width>110</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Program Win Count</string>
    </property>
   </widget>
   <widget class="QLabel" name="playerWinCountLbl">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>60</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="programWinCountLbl">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>60</y>
      <width>110</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
